rpm/python-ramalama.spec

```diff
@@ -47,13 +47,8 @@ configure the system for AI themselves. After the initialization, RamaLama
 will run the AI Models within a container based on the OCI image.
 
 %package -n python%{python3_pkgversion}-%{pypi_name}
-Requires: podman
-%if 0%{?fedora} >= 40
-# Needed as seen by BZ: 2327515
-Requires: python%{python3_pkgversion}-omlmd
-%else
+Recommends: podman
 Recommends: python%{python3_pkgversion}-omlmd
```

Comment:
We could actually remove this altogether potentially

Reply:
If you remove it the first time you pull from Ollama it is going to complain about it not being installed.

Reply:
This isn't used for pulling from Ollama, the Ollama puller is written from scratch in python. What omlmd was doing before was a wrapper for oras and tried to pull OCI artefacts using oras.

Reply:
Ok I see self._pull_omlmd(args) which looks wrong, it is only called in oci.  We really should not call it there, and then we can remove all of the code.